game.cpp

```diff
@@ -761,6 +762,23 @@ void place_nth_food_randomly(game &g, const int &n)
   g.get_food()[n].place_randomly(g.get_rng(), {get_min_x(g), get_min_y(g)}, {get_max_x(g), get_max_y(g)});
 }
 
+coordinate get_nth_shelter_position(const game &g, const int &n)
+{
+  return g.get_shelters()[n].get_position();
+}
+
+/// Get all shelter positions
+std::vector<coordinate> get_all_shelter_positions(const game& g)
```

Comment:
Nice! One day, you'll learn algorithms and replace that 'dumb' for-loop by such an algorith, For now: well done!